docs/docs/reference/new-types/match-types.md

```diff
@@ -143,55 +143,6 @@ The third rule states that a match type conforms to its upper bound
 
 Within a match type `Match(S, Cs) <: B`, all occurrences of type variables count as covariant. By the nature of the cases `Ci` this means that occurrences in pattern position are contravarant (since patterns are represented as function type arguments).
 
-## Handling Termination
-
```

Comment:
Agree that the tone of the description is more like an issue and not like a spec. But a spec still has to talk about termination. So the info needs to be given, just formulated differently.